shared/macros-oval.jinja

```diff
@@ -844,8 +844,8 @@
 {{%- set sshd_config_path = "/etc/ssh/sshd_config" %}}
 {{%- set sshd_config_dir = "/etc/ssh/sshd_config.d" -%}}
 {{%- set description = "Ensure '" ~ parameter ~ "' is configured with value '" ~ value ~ "' in " ~ sshd_config_path %}}
-{{%- if config_is_distributed %}}
-{{%- set description = description  ~ " and in " ~ sshd_config_dir -%}}
```

Comment:
@matejak I'd appreciate your review on the description variable above, I believe it should say:
`Ensure parameter '<parameter>' is configured with value '<value>' in /etc/ssh/sshd_config or in /etc/ssh/sshd_config.d`

Also, any fedback on the `sshd_distributed_config` parametrization is welcome.